resources/lang/en/accounts.php

```diff
@@ -47,7 +47,7 @@
             'user' => [
                 'user_from' => 'current location',
                 'user_interests' => 'interest',
-                'user_msnm' => 'skype',
```

Comment:
where should I change osu! -> osu!next ?

*******

will use "Twitter" and "Interest" later